### PR TITLE
typo: longson should be loongson

### PR DESCRIPTION
**Pull request by linkslice**

### CODE REVIEW

File: Completion/BSD/Type/_obsd_architectures [hunk 1]
```diff
@@ -3,4 +3,4 @@
 local expl
 
 _description architectures expl 'architecture'
-compadd "$@" "$expl[@]" alpha amd64 arm64 armv7 hppa i386 landisk longson luna88k macppc octeon sgi sparc64
+compadd "$@" "$expl[@]" alpha amd64 arm64 armv7 hppa i386 landisk loongson luna88k macppc octeon sgi sparc64
```
Comment (danielshahaf) [inline Completion/BSD/Type/_obsd_architectures]: Should `sgi` be retained or removed? I don't see it under https://github.com/zsh-users/zsh/blob/243083c50b7cb8f7b1539abcf8418ebb3163865e/Completion/BSD/Type/_obsd_architectures#L6.

Comment (danielshahaf) [reply Completion/BSD/Type/_obsd_architectures]: ... under https://www.openbsd.org/faq/faq4.html#Download.

**Comment by danielshahaf** [comment]
Merged in 2f419b332d9cf28eaad0523b3404ab8ce52a82e0 with a tweaked log message.  Let us know about `sgi`.  Thanks for the patch!

File: Completion/BSD/Type/_obsd_architectures (hunk 1 above)
Comment (linkslice) [reply Completion/BSD/Type/_obsd_architectures]: probably not. Support for was dropped with the latest release. https://www.openbsd.org/sgi.html

Comment (danielshahaf) [reply Completion/BSD/Type/_obsd_architectures]: Thanks, done in dd50f125b5eb65896642d2ff664adefd33f1004c.

**Pull request closed without merging**